api/src/test/java/com/sanction/thunder/models/EmailTest.java

```diff
@@ -24,44 +20,44 @@ public void testToJson() throws Exception {
     String expected = MAPPER.writeValueAsString(
         MAPPER.readValue(FixtureHelpers.fixture("fixtures/email.json"), Email.class));
 
-    assertEquals(expected, MAPPER.writeValueAsString(email));
+    Assertions.assertEquals(expected, MAPPER.writeValueAsString(email));
```

Comment:
Can you please use the `static` import for these in all of the classes? It's convention to use static import for Junit methods because it reduces the verbosity and its more clear.

https://stackoverflow.com/questions/2791952/should-i-use-static-import

Reply:
Then most of these lines won't change, it will just be the imports that change:

```java
import static org.junit.Assert.assertEquals;		
import static org.junit.Assert.assertFalse;		
import static org.junit.Assert.assertNotEquals;		
import static org.junit.Assert.assertTrue;
```

instead of:

```java
import org.junit.jupiter.api.Assertions;
```